project/src/components/fiim-overview/film-overview.tsx

```diff
@@ -0,0 +1,51 @@
+import { Fragment } from 'react';
+import { useFilm } from '../../hooks/use-film';
+import NotFound from '../404-screen/404-screen';
+
+function FilmOverview(): JSX.Element {
+  const film = useFilm();
+  if(typeof film === 'undefined'){
+    return <NotFound />;
+  }
+  const {rating,director, description, starring, scoresCount} = film;
+  return(
+    <>
+      <div className="film-rating">
+        <div className="film-rating__score">{rating}</div>
+        <p className="film-rating__meta">
+          <span className="film-rating__level">Very good</span>
+          <span className="film-rating__count">{scoresCount} ratings</span>
+        </p>
+      </div>
+
+      <div className="film-card__text">
+
+        <p>{description}</p>
+
+        <p className="film-card__director"><strong>Director: {director}</strong></p>
+
+        <p className="film-card__starring">
+          <strong>
+            Starring:
+            {
+              starring.map((el) => (
+                <Fragment key={`${el}`}>
```

Comment:
а так? пять лишних букв!!! :)
```suggestion
                <Fragment key={el}>
```